lib/scsi_defs.h

```diff
@@ -3,8 +3,14 @@
 
 #include <cstdint>
 
+#include "absl/base/attributes.h"
+#include "absl/base/casts.h"
+
+
 namespace scsi_defs {
 
+  const uint32_t kInquiryCommandSize = 6;
```

Comment:
Prefer `constexpr` for POD global constants like this.

Reply:
Good suggestion. I updated the code to use `constexpr`